src/components/ChatEntry.js

```diff
@@ -1,22 +1,46 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+import { useState } from 'react';
+
+
+const ChatEntry = ({sender, body, id, timeStamp, liked, updateChatData}) => {
```

Comment:
Remember that all of these are coming in from the parent component as props, so you could replace this entire dictionary with "props" and then access each element as "props.sender" or "props.body", etc.